test/personnummer.dart

```diff
@@ -19,7 +19,7 @@ void main() {
   Personnummer.dateTimeNow = new DateTime(2019, 7, 13);
```

Comment:
If you could fetch the data at test-time from the meta package, that would be great (so that test-data could be updated without having to update tests). It's okay like this though, no big deal! :)

Reply:
I'll update the test cases later with the new testdata. Not a blocking issue :)